samples/Nancy.Swagger.Annotations.Demo/Startup.cs

```diff
@@ -29,7 +31,33 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseOwin(x => x.UseNancy());
```

Comment:
In my use case, I want to leave the Bootstrapper class alone, because it is used at runtime of the Nancy web server. I want to have a clean cut here between the bootstrapper used for swagger.json generation and the production bootstrapper.